gis/qgis3/Portfile

```diff
@@ -26,23 +26,23 @@ subport ${name-ltr} {}
 
 if {${subport} eq ${name}} {
     # Latest version
-    github.setup    qgis QGIS 3_32_1 final-
+    github.setup    qgis QGIS 3_32_2 final-
     revision        1
```

Comment:
the revision should be reset to 0 when updating the version

Reply:
Missed these ones, fixed. Thanks!